packages/FormElement/README.md

```diff
@@ -22,107 +22,172 @@ npm install @paprika/form-element
+| children | node            | false    | null    | Content for left column flex child (typically the Label)                    |
+| width    | [number,string] | false    | "auto"  | Width of the left column as number (pixels) or string (any CSS width value) |
+
+### FormElement.Layout.RightCol
+
+| Prop     | Type | required | default | Description                                                                 |
+| -------- | ---- | -------- | ------- | --------------------------------------------------------------------------- |
+| children | node | false    | null    | Content for right column flex child (typically everything except the Label) |
+
+### Fieldset
+
+Props are the same as `FormElement` but `hasFieldSet=true`.
+
 ## Usage
 
-Please use `<L10n />` component to wrap `<FormElement />` or you application.
+The `<L10n />` component is a required context provider that must wrap the `<FormElement />` (or your application) for proper localization.
 
-Using with paprika `<Input />`
+### Using with `<Input />`
 
+<!-- prettier-ignore -->
 ```jsx
-import FormElement from "@paprika/form-element";
 import Input from "@paprika/input";
+import FormElement from "@paprika/form-element";
+
+const { Label, Content } = FormElement;
 
 <FormElement>
-  <Label>Name</Label>
+  <Label>First Name</Label>
   <Content>
     {a11yProps => (
       <Input
         onChange={handleChange}
         value={value}
-        placeholder="Form placeholder"
-        aria-required={hasRequiredLabel}
-        hasError={false}
-        isDisabled={isDisabled}
-        isReadOnly={isReadOnly}
-        size={size}
         {...a11yProps}
       />
     )}
-    )}
   </Content>
-</FormElement>;
+</FormElement>
 ```
 
-Using with paprika `<DatePicker />`
+### Using with `help` popover, `<Instructions />`, `<Description />`, and `<Error />`
+
+Best practices for laying out supporting content for a form input is to follow this order:
 
+- `<Label />` first.
+- `<Instructions />` above the input field contained in `<Content />`.
+- `<Description />` or `<Error />` shown below input field (but do not render both at the same time).
+
+Note: the `<Error />` component will render `null` if its `children` is falsey.
+
+<!-- prettier-ignore -->
 ```jsx
-import DatePicker from "@paprika/date-picker";
-import FormElement, { Label, Content } from "@paprika/form-element";
+import Input from "@paprika/input";
+import FormElement from "@paprika/form-element";
+
+const { Label, Content, Instructions, Description, Error } = FormElement;
+const errorMsg = "There was an error.";
 
 <FormElement>
-  <Label>Name</Label>
+  <Label help="Help text">Field label</Label>
+  <Instructions>Instruction text</Instructions>
+  <Content>
+    {a11yProps => (
+      <Input hasError={Boolean(errorMsg)} {...a11yProps} />
+    )}
+  </Content>
+  {errorMsg ? <Description>Description text</Description> : null}
+  <Error>{errorMsg}</Error>
```

Comment:
Yes, should be `!errorMsg` 🤦 

My thinking was that since `<Error>` already [returns `null` when its `children` are falsey](https://github.com/acl-services/paprika/pull/835/files#diff-f984d98b1cdde18b6eac74859149f5629445adb2a89cded3a57de9c28c3c0197R16), it's not necessary to conditionally render it... but your suggestion doesn't hurt anything and I agree it _IS_ clearer.  I'll change it.